pikaur/news.py

```diff
@@ -0,0 +1,151 @@
+import datetime
+import urllib.request
+import urllib.error
+import xml.etree.ElementTree
+import os
+from http.client import HTTPResponse
+from html.parser import HTMLParser
+
+from typing import TextIO
+
+from pikaur.config import CACHE_ROOT
+import pikaur.pprint
```

Comment:
from pikaur.pprint import print_stdout, format_paragraph

Reply:
Done. 3407edd

Reply:
oops, `from .pprint import....`

Reply:
btw you can run the check locally, `./maintenance_scripts/lint.sh`

Reply:
The imports will be fixed in the last step, have some issues with my IDE.